src/store/index.ts

```diff
@@ -90,7 +88,10 @@ export const _hydrationReducer: typeof rootReducer = (state, action) => {
 const makeStore = (initialState?: Record<string, any>) => {
   return configureStore({
     reducer: _hydrationReducer,
-    middleware: (getDefaultMiddleware) => getDefaultMiddleware({ serializableCheck: false }).concat(middleware),
+    middleware: (getDefaultMiddleware) => {
+      listeners.forEach((listener) => listener(listenerMiddlewareInstance))
```

Comment:
Is there a difference in registering the listeners in this function or on the top level? In the examples from the documentation they are registered on the top level right after the middleware is created.

Reply:
I don't think so.

I moved it here as this is just before store creation/allowed for better testing. What are your thoughts?